Configuration/AlCa/python/autoCondPhase2.py

```diff
@@ -0,0 +1,70 @@
+# default value for all customizations
+connectionString = "frontier://FrontierProd/CMS_CONDITIONS"
+
+SiPixelLARecord           =   "SiPixelLorentzAngleRcd"                  
+SiPixelSimLARecord        =   "SiPixelLorentzAngleSimRcd"   
+SiPixelGenErrorRecord     =   "SiPixelGenErrorDBObjectRcd"       
+SiPixelTemplatesRecord    =   "SiPixelTemplateDBObjectRcd"       
+SiPixel2DTemplatesRecord  =   "SiPixel2DTemplateDBObjectRcd"     
+
+LA = {
```

Comment:
These individual dicts could be combined into a single dict of dicts, with a list of actively used keys. Then the `phase2GTs` dict could be filled automatically from the active list. (The list of active subdet names could also be provided for this purpose.)

Reply:
For our case, the nice thing of keeping the dictionaries separate is that at the moment we are not yet using the full list of possible customizable records, but only the Lorentz Angle related ones. So one can piece by piece increase the number of "customizations" by adding more payload to each of the  `phase2GTs` keys. 
Did I miss your point? Would you care to suggest the structure of this dict of dicts?

Reply:
This is covered by part of my suggestion: "a list of actively used keys". For example:
```python
activeKeys = ["LA","LAWidth","SimLA"]
activeDets = ["T6","T14","T15"]
phase2GTs = {}
for det in activeDets:
    appendedTags = ()
    for key in activeKeys:
       appendedTags += allTags[key][det]
    phase2GTs["phase2_realistic_"+det] = ('phase2_realistic', appendedTags)
```

Reply:
done few lines below.